pkg/universe.dagger.io/x/paul@fgtech.fr/ansible/playbook.cue

```diff
@@ -0,0 +1,110 @@
+package ansible
+
+import (
+	"strings"
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#SSHKeys: {
+	// Folder containing ssh keys to mount [required]
+	source: dagger.#FS
+
+	// Folder path where to mount keys in the running container (ie: the one referenced in the Playbook configuration) [default="/root/.ssh"]
+	destPath: string | *"/root/.ssh"
+}
+
+#Inventory: {
+	// Folder where the inventory file is located [required]
+	source: dagger.#FS
+
+	// Name of the inventory file [required]
+	file: string
+}
+
+#PlaybookSource: {
+	// Folder containing the ansible project sources (with tasks, vars, etc.) [required]
+	source: dagger.#FS
+
+	// Main playbook relative path to run in the project directory [default="main.yml"]
+	mainPlaybook: string | *"main.yml"
+
+	// Output path of the playbook generated files [default = "./output"]
+	outputPath: string | *"/output"
+}
+
+#Playbook: {
+	// Ansible Config for running the playbook [required]
+	configImage: docker.#Image
+
+	// Inventory to use for the playbook [required]
+	inventory?: #Inventory
+
+	// Playbook to run [required]
+	playbookSource: #PlaybookSource
+
+	// Struct of the type { "key": "value" }
+	extraVars: [string] | *[]
```

Comment:
Currently `[string]` is not a struct, that should be `[string]: string`

Reply:
Actually it is treated as a list next, the documentation was not up to date :)